src/test/java/com/wikia/webdriver/common/dataprovider/MercurySEODataProvider.java

```diff
@@ -0,0 +1,48 @@
+package com.wikia.webdriver.common.dataprovider;
+
+import com.wikia.webdriver.common.core.url.Page;
+import org.testng.annotations.DataProvider;
+
+public class MercurySEODataProvider {
+
+    @DataProvider
+    public static Object[][] htmlTitles() {
+        return new Object[][] {
+            {
+                new Page("sktest123", "/wiki/Sktest123_Wiki"), "Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Style-5H2"), "Style-5H2 - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/TestDisplayTitle"), "testing abc - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Category:Premium_Videos"), "Category:Premium Videos - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Category:Non-premium_Videos"), "Category:Non-premium Videos - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("sktest123", "/wiki/Category:Premium"), "PremiumVideos - Sktest123 Wiki - Wikia"
+            }, {
+                new Page("es.pokemon", "/wiki/WikiDex"), "WikiDex - Wikia"
+            }, {
+                new Page("es.pokemon", "/wiki/Lista_de_Pokémon"), "Lista de Pokémon - WikiDex - Wikia"
+            }, {
+                new Page("es.pokemon", "/wiki/Categoría:Regiones"), "Categoría:Regiones - WikiDex - Wikia"
+            }, {
+                new Page("starwars", "/wiki/Main_Page"), "Wookieepedia - Wikia"
+            }, {
+                new Page("starwars", "/wiki/Droid_starfighter"), "Droid starfighter - Wookieepedia - Wikia"
+            }, {
+                new Page("starwars", "/main/category/Future_films"), "Future_films"
+            }, {
+                new Page("starwars", "/main/section/Films"), "Films"
+            }, {
+                new Page("mediawiki119", "/d/f/203236/latest"), ""
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "mediawiki119" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)